docs/accessibility/results-api.mdx

```diff
@@ -410,5 +411,38 @@ workflows:
       - run-cypress
 ```
 
+</TabItem>
+
+<TabItem value="AWS CodeBuild">
+
+```diff
```

Comment:
can add a title if the name of this file is normalized like:

```
title="test_cypress.yaml"
```

Reply:
nice - looks like `buildspec.yaml` is the convention for CodeBuild